src/racer/config.rs

```diff
@@ -0,0 +1,23 @@
+use std::path::PathBuf;
```

Comment:
Having a generic `config.rs` makes me suspicious; it seems likely that it'll become a parking lot for a bunch of unrelated items. Can you make the name for this more specific to file paths?

Reply:
Yes, I can; however, I originally thought that it would be actually good to have a centralised module for reading any needed configuration from the environment and keeping the state. Do you think it would make sense to move this to `nameres.rs` where `RUST_SRC_PATHS` is used?

Reply:
I'm personally a fan of smaller, purpose-oriented modules, so I'd suggest one that only contained this and have that be used by `nameres`. However, I'm fine with it living in `nameres.rs` directly for the time-being.

Reply:
All right, I'll refactor that.